lib/fast_versioning/engine.rb

```diff
@@ -1,5 +1,7 @@
 module FastVersioning
   class Engine < ::Rails::Engine
     isolate_namespace FastVersioning
+
+    PaperTrail.config.track_associations = false
```

Comment:
I don't think we should change any PaperTrail configuration in the engine, that should be done in the application - why is that needed here?

Reply:
Thats what I thought too, but the specs were throwing warnings about this until I put this in here, even after configuring this in the setup of the dummy app for the spec.